tests/components/dsmr/test_sensor.py

```diff
@@ -140,7 +141,7 @@ def mock_connection_factory(monkeypatch):
         abs(entity.state - 0.033) < 0.00001
     ), "state should be hourly usage calculated from first and second update"
 
-    assert entity.unit_of_measurement == "m3/h"
+    assert entity.unit_of_measurement == f"m3/{TIME_HOURS}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)